### DISCUSSION TITLE
How to play most recent episode in show?

### DISCUSSION THREAD

**Discussion by rimago** [librespot-org/librespot-java#682] [Q&A]
Hi community,

when I call play with an uri of a show, the oldest episode is played. How can I play the most recent episode?

Background:
I have a spotify show with podcasts and would like to always play the newest podcast first.

Any support and hints are appreciated.

**Comment by tobiasguyer** [comment]
i don't know how that call would look in java, but you just have to call for the last object of the show 
(the episodes are orderd in historical order with their first episode as first object)
